docs/detectors/nuget.md

```diff
@@ -0,0 +1,15 @@
+# NuGet Detection
+
+## Requirements
+
+NuGet Detection depends on the following to successfully run: 
+
+- One or more `*.nuspec`, `*.nupkg`, `*.packages.config`, or `.*csproj` files.
+
+## Detection Strategy 
+
+NuGet Detection is performed by parsing any `*.nuspec`, `*.nupkg`, `*.packages.config`, or `*.project.assets` files found under the scan directory. By searching for all `*.nuspec,` `*.nupkg` files on disk the global NuGet cache gets searched which can include packages that are not included in the final build.
+
+## Known Limitations
+
+Currently the NuGet detector is over-reporting because the global NuGet cache gets searched. To solve this over-reporting a new NuGet Detector approach will be rolled out. This new approach will now only parse `*.packages.config`, or `*.project.assets` (`*.csproj`) files. So any components that are only found in `*.nuspec,` or `*.nupkg` files will not be detected with the new NuGet Detector approach.
```

Comment:
Again, break this down. `NuGet` detector is overreporting. I would also specify that this is because of NuGet's restore behaviour[^1] which downloads all possible dependencies, before [resolving the final dependency graph][1].

Side note: I think using [NuGet lockfiles][2] might prevent this behaviour 🤔 

[1]: https://learn.microsoft.com/en-us/nuget/concepts/dependency-resolution
[2]: https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#enabling-the-lock-file
[^1]: https://learn.microsoft.com/en-us/nuget/consume-packages/package-restore#package-restore-behavior